nixos/doc/manual/installation/installing-usb.section.md

```diff
@@ -1,31 +1,54 @@
-# Booting from a USB Drive {#sec-booting-from-usb}
+# Booting from a USB flash drive {#sec-booting-from-usb}
 
-For systems without CD drive, the NixOS live CD can be booted from a USB
-stick. You can use the `dd` utility to write the image:
-`dd if=path-to-image of=/dev/sdX`. Be careful about specifying the correct
-drive; you can use the `lsblk` command to get a list of block devices.
+The image has to be written verbatim to the USB flash drive for it to be
+bootable on UEFI and BIOS systems. Here are the recommended tools to do that.
 
-::: {.note}
-::: {.title}
-On macOS
+## Creating bootable USB flash drive with a graphical tool {#sec-booting-from-usb-graphical}
+
+Etcher is a popular and user-friendly tool. It works on Linux, Windows and macOS.
+
+Download it from [balena.io](https://www.balena.io/etcher/), start the program,
+select the downloaded NixOS ISO, then select the USB flash drive and flash it.
+
+::: {.warning}
+Etcher reports errors and usage statistics by default, which can be disabled in
+the settings.
 :::
 
+An alternative is [USBImager](https://gitlab.com/bztsrc/usbimager/#usbimager),
+which is very simple and does not connect to the internet. Download the version
+with write-only (wo) interface for your system. Start the program,
+select the image, select the USB flash drive and click "Write".
+
+## Creating bootable USB flash drive from a Terminal on Linux {#sec-booting-from-usb-linux}
+
+First, make sure all partitions on the device are properly unmounted. Then use
+the `dd` utility to write the image to the USB flash drive:
+
+```ShellSession
+sudo umount /dev/sdX*
+sudo dd if=<path-to-image> of=/dev/sdX bs=4M
+```
+
+Be careful to replace `sdX` with the correct device! You can use the `lsblk`
```

Comment:
Users might read this to late :) Maybe lsblk should be moved up into the example?

Reply:
Maybe some example output and than what device was chosen:

```
$ lsblk
NAME        MAJ:MIN RM   SIZE RO TYPE MOUNTPOINTS
nvme0n1     259:0    0   1,8T  0 disk
├─nvme0n1p1 259:1    0  1000M  0 part /boot
├─nvme0n1p2 259:2    0    16M  0 part
├─nvme0n1p3 259:3    0 195,3G  0 part
└─nvme0n1p4 259:4    0   1,6T  0 part
```

Reply:
not sure how to move it up. having the output there would add a lot of clutter

should we just start with the task to find the right disk and explain the process in more detail?

on my system:

```
[davidak@gaming:~/code/nixpkgs]$ lsblk
NAME        MAJ:MIN RM   SIZE RO TYPE MOUNTPOINTS
sda           8:0    0 465.8G  0 disk 
├─sda1        8:1    0     8G  0 part [SWAP]
└─sda2        8:2    0 457.8G  0 part /nix/store
                                      /
sdb           8:16   1   3.7G  0 disk 
├─sdb1        8:17   1   2.2G  0 part 
└─sdb2        8:18   1     3M  0 part 
nvme0n1     259:0    0 931.5G  0 disk 
├─nvme0n1p1 259:1    0   512M  0 part 
├─nvme0n1p2 259:2    0 896.6G  0 part 
└─nvme0n1p3 259:3    0  34.4G  0 part
```

Reply:
Maybe this could be just:

```
# This command will print all disks, identify your usb stick device name from this list and use it in the commands below.
# WARNING: if you pick the wrong device you may override important data.
$ lsblk 
```

Reply:
i will add instructions to find the correct device as first step.

it's quiet unlikely that someone has `/dev/sdX` AND don't know what they are doing (just executing the command without understanding), but when someone just runs the command and get an error, our instructions are bad, so we should improve the experience

thanks for the suggestion!

Reply:
I have rewritten the complete sections in numbered list style.